tests/monitor/monitor.test.mjs

```diff
@@ -372,230 +396,187 @@ describe('setup', () => {
+              );
+              assert(
+                gateway.operatorStake >= 10_000_000_000,
+                `Gateway ${gateway.gatewayAddress} has less than 10_000_000_000 ARIO staked`,
+              );
+              assert(
+                gateway.stats.failedConsecutiveEpochs >= 0,
+                `Gateway ${gateway.gatewayAddress} has less than 0 failed consecutive epochs`,
+              );
+              assert(
+                gateway.stats.failedConsecutiveEpochs < 30,
+                `Gateway ${gateway.gatewayAddress} has more than 30 failed consecutive epochs`,
+              );
+              assert(
+                gateway.stats.passedConsecutiveEpochs <= currentEpochIndex,
+                `Gateway ${gateway.gatewayAddress} has more passed consecutive epochs than current epoch index`,
+              );
+              assert(
+                gateway.stats.passedConsecutiveEpochs >= 0,
+                `Gateway ${gateway.gatewayAddress} has less than 0 passed consecutive epochs`,
+              );
+              assert(
+                gateway.stats.totalEpochCount <= currentEpochIndex,
+                `Gateway ${gateway.gatewayAddress} has more total epochs than current epoch index`,
+              );
+              assert(
+                gateway.stats.totalEpochCount >= 0,
+                `Gateway ${gateway.gatewayAddress} has less than 0 total epochs`,
+              );
+              assert(
+                gateway.stats.prescribedEpochCount <= currentEpochIndex,
+                `Gateway ${gateway.gatewayAddress} has more prescribed epochs than current epoch index`,
+              );
+              assert(
+                gateway.stats.prescribedEpochCount >= 0,
+                `Gateway ${gateway.gatewayAddress} has less than 0 prescribed epochs`,
+              );
+            }
+            if (gateway.status === 'leaving') {
+              assert(gateway.totalDelegatedStake === 0);
+              assert(gateway.operatorStake === 0);
+            }
+          }),
+        ),
+      );
       assert(
         uniqueGateways.size === totalGateways,
         `Counted total gateways (${uniqueGateways.size}) does not match total gateways (${totalGateways})`,
       );
     });
 
     it('should have valid delegates for all gateways', async () => {
-      const { items: gateways } = await io.getGateways({
-        limit: 1000,
+      const { items: delegates } = await io.getAllDelegates({
+        limit: 10_000,
       });
-      // NOTE: this is an attempt to get the delegates for a gateway, the UX is not great so consider modifying or adding handlers to support gateway specific delegation filters
-      for (const gateway of gateways) {
-        const { items: delegates } = await io.getGatewayDelegates({
-          address: gateway.gatewayAddress,
-        });
-        if (delegates.length > 0) {
-          for (const delegate of delegates) {
+      // this overwhelms the local CU, so keep in a for loop
```

Comment:
Was this a reminder to go back to the loop?

Reply:
no i will remove - i changed the test assertion given the new API

Reply:
[ef52d7b](https://github.com/ar-io/ar-io-network-process/pull/330/commits/ef52d7bf91559dc4aaaefe6c9ef3752681cec4ce)